packages/lab/src/dialog-next/DialogNextContext.tsx

```diff
@@ -0,0 +1,13 @@
+import { createContext, useContext } from "react";
```

Comment:
Why do we need a Context Provider for this component ?

Reply:
I've flipped-flopped on this a bit. In this iteration I mainly needed it to pass dialog id (in order to connect aria attributes), and ref to autofocus mainly to dialog title component. The reason I need a context is because I don't know what will get passed as children to Dialog. The user can nest the children as many times as they want. For example
```tsx
<Dialog>
  <StackLayout>
    <DialogTitle>Title</DialogTitle>
    <DialogContent>...</DialogContent>
  </StackLayout>
</Dialog>
```
So I cannot clone children.

Reply:
In the next iteration of design it looks like I will also need to pass the status prop value as well to the title comp since it will probably render the status icon.